google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionProperties.java

```diff
@@ -257,6 +259,13 @@ class ConnectionProperties {
           DEFAULT_ROUTE_TO_LEADER,
           BooleanConverter.INSTANCE,
           Context.STARTUP);
+  static final ConnectionProperty<Boolean> ENABLE_END_TO_END_TRACING =
```

Comment:
here also: Group with the other tracing properties and modify the description (the list in `ConnectionOptions` will be removed soon, we're in the middle of a refactor for these properties)

Reply:
Moved and updated the description. Please review